Modules/System/Base64 Convert/src/Base64ConvertImpl.Codeunit.al

```diff
@@ -31,6 +31,55 @@ codeunit 4111 "Base64 Convert Impl."
         exit(Base64String);
     end;
 
+    procedure ToBase64(String: Text; TextEncoding: TextEncoding): Text
+    begin
+        exit(ToBase64(String, false, TextEncoding, 0));
+    end;
+
+    procedure ToBase64(String: Text; TextEncoding: TextEncoding; Codepage: Integer): Text
+    begin
+        exit(ToBase64(String, false, TextEncoding, Codepage));
+    end;
+
+    procedure ToBase64(String: Text; InsertLineBreaks: Boolean; TextEncoding: TextEncoding): Text
+    begin
+        exit(ToBase64(String, InsertLineBreaks, TextEncoding, 0));
+    end;
+
+    procedure ToBase64(String: Text; InsertLineBreaks: Boolean; TextEncoding: TextEncoding; Codepage: Integer): Text
+    var
+        Convert: DotNet Convert;
+        Encoding: DotNet Encoding;
+        Base64FormattingOptions: DotNet Base64FormattingOptions;
+        Base64String: Text;
+    begin
+        if String = '' then
+            exit('');
+
```

Comment:
We can reduce the the if case code duplication by 

if InsertLineBreaks then
   Base64FormattingOptions := Base64FormattingOptions.InsertLineBreaks
else
   Base64FormattingOptions := Base64FormattingOptions.None;

Reply:
Done